HardCodedStringCheckerSharp/AppController.cs

```diff
@@ -77,6 +78,8 @@ private bool MakeFixesOnFile( string file, Action eAction )
             return false;
          if ( fileName.ToLower().Contains( ".g." ) )
             return false;
+         if ( fileName.ToLower().Contains( "test" ) )
```

Comment:
Why not `EndsWith`?  I think that's already the convention for most of our files?  The word `test` *could* (though admittedly unlikely) appear in the middle of legitimate words that we'd want the checker on.